### DISCUSSION TITLE
What do you use Age for?

### DISCUSSION THREAD

**Discussion by proyb6** [FiloSottile/age#321] [Q&A]
File encryption is useful for many reasons to encrypt sensitive contents over the web or offline storage, what do you use Age for in real world?

**Comment by technion** [comment]
I recently worked to replicate the signing CI I proposed for Age, on my own project. The difficulty is that I'm signing Powershell, and the only tooling to do that was.. very messy. One aspect is that it required a binary file.

In the end, the nicest way to bundle my certificates was to Age encrypt them with -a, and place the decryption key in a Github secret. You can find some horrible Powershell here that downloads Age and powers it all:

https://github.com/technion/open_safety/blob/main/.github/workflows/release.yml

**Comment by AnimMouse** [comment]
Encrypting .tar files.

**Comment by cyb3rz3us** [comment]
Like many others, I use Age to encrypt anything that is being put into cold cloud storage.  Very easy to integrate into my automation workflows for both directions of the encrypt\decrypt routines.
